### PR TITLE
[FIX] prevent entering a blank amount_currency in an account.move.line

### PR DESCRIPTION
**Pull request by veloutin**
This is a fix for https://github.com/odoo/odoo/issues/4242 
Upstream PR: https://github.com/odoo/odoo/pull/4457

### CODE REVIEW

File: addons/account/account_move_line.py [hunk 1]
```diff
@@ -609,8 +612,16 @@ def _check_currency(self, cr, uid, ids, context=None):
         return True
 
     def _check_currency_and_amount(self, cr, uid, ids, context=None):
+        digits = self.pool['decimal.precision'].precision_get(
+            cr, SUPERUSER_ID, 'Account')
```
Comment (pedrobaeza) [inline addons/account/account_move_line.py:616]: Why do you need SUPERUSER_ID?

Comment (veloutin) [inline addons/account/account_move_line.py:616]: That's how it is done in the decimal precision module: https://github.com/OCA/OCB/blob/8.0/addons/decimal_precision/decimal_precision.py#L74

I do not know why it is like that in there either :)

**Comment by pedrobaeza** [comment]
:+1:

**Comment by veloutin** [comment]
This breaks some account_voucher tests at the moment.

**Comment by pedrobaeza** [comment]
Closing as in upstream

**Pull request closed without merging**